lib/reporters/xunit.js

```diff
@@ -130,7 +130,7 @@ XUnit.prototype.test = function(test) {
 
   if (test.state === 'failed') {
     var err = test.err;
-    this.write(tag('testcase', attrs, false, tag('failure', {}, false, cdata(escape(err.message) + '\n' + err.stack))));
```

Comment:
* No idea, ask @jkrall 
* CDATA is for humans to type content into XML, entity escaping is how XML libs do it.